src/scripts/dom.js

```diff
@@ -262,6 +262,50 @@
         return _transitionEvent;
     }
 
+    /**
+     * Gets the value of a parameter in the URL.
+     * @param {*} name Name of the parameter to get.
+     * @param {*} url URL used to get the parameter.
+     */
+    export function getParameterByName(name, url = window.location.href) {
+        const urlObject = new URL(url);
+        const params = new URLSearchParams(urlObject.search);
+
+        return params.get(name);
+    }
```

Comment:
This in particular really needs to be checked. The original is in site.js.

Reply:
```
url: http://.../jellyfin/web/index.html#!/movies.html?topParentId=b853ad1780b36e82af77f6d2445b7c22
urlObject.search: ''
hash: '#!/movies.html?topParentId=b853ad1780b36e82af77f6d2445b7c22'
```
`params.get('topParentId'): null`

`new URLSearchParams('topParentId=b853ad1780b36e82af77f6d2445b7c22').get('topParentId'): 'b853ad1780b36e82af77f6d2445b7c22'`

Maybe we should just cut of `*?`? `url.replace(/.*\?/, '')`